website/src/docs/deprecations/version-2-schema.md

```diff
@@ -1,17 +1,17 @@
 ---
-slug: /deprecations/version-2-schema/
+title: "Version 2 Schema (#1197)"
+description: Dépréciation du schéma version 2 des Taskfiles et migration vers la version 3
```

Comment:
en anglais s'il vous plaît.

Same in:

- `/deprecations/completion-scripts.md`
- `/deprecations/index.md`
- `/deprecations/template-functions.md`
- `/deprecations/template.md`
- `/deprecations/version-2-schema.md`
- `/experiments/env-precedence.md`
- `/experiments/gentle-force.md`
- `/experiments/index.md`
- `/experiments/remote-taskfiles.md`

Reply:
- `/deprecations/completion-scripts.md` :heavy_check_mark: 
- `/deprecations/index.md` :heavy_check_mark: 
- `/deprecations/template-functions.md` :heavy_check_mark: 
- `/deprecations/template.md` :heavy_check_mark: 
- `/deprecations/version-2-schema.md` :heavy_check_mark: 
- `/experiments/env-precedence.md`  :heavy_check_mark: 
- `/experiments/gentle-force.md`  :heavy_check_mark: 
- `/experiments/index.md`  :heavy_check_mark: 
- `/experiments/remote-taskfiles.md`  :heavy_check_mark: